cloudformation/jolly-roger.yaml

```diff
@@ -522,6 +555,27 @@ Resources:
                         "max-size": "100m"
                       }
                     }
+                - path: /usr/local/bin/renew-certificate
+                  content: |
+                    #!/bin/bash
+                    set -e
+
+                    # Renew the certificate and update haproxy's offline copy.
+                    if [ -n "${CertNotificationEmail}" ]; then
+                      email_flag="-m ${CertNotificationEmail}"
+                    else
+                      email_flag="--register-unsafely-without-email"
+                    fi
+                    certbot -v certonly --dns-route53 -d ${AppUrl} --non-interactive --agree-tos $email_flag
```

Comment:
Does this only renew certs that are close to expiry, or will it forcibly renew the certificate daily? (It looks like `certbot renew` is intelligent, but I'm having a hard time telling from the docs if that's true of `certonly` also)

Reply:
Same behavior as renew - it will keep the certificate until the cert is within 30 days of expiration. (Just ran the script manually on my instance to double check).